### PR TITLE
fix: cancellation email for seated event for 2nd attendee onwards

### LINKED ISSUE #10167: [CAL-2175] No cancellation email when canceling seat
### Issue Summary

When a seat is canceled the organizer an the attendee are not receiving a cancellation email. It only works for the first seat. 

### Steps to Reproduce

1. Create a seated event 
2. Book a seat 
3. Book another seat in the same slot 
4. Cancel the second seat 
5. See that no cancelation email is sent

<sub>[CAL-2175](https://linear.app/calcom/issue/CAL-2175/no-cancellation-email-when-canceling-seat)</sub>

### PR DESCRIPTION
**Pull request by iamr-kumar**
## What does this PR do?

Fixes no cancellation email for seated event when attendee >= 2

Fixes #10167 

## Type of change

  - [x] Bug fix (non-breaking change which fixes an issue)
  - [x] Chore (refactoring code, technical debt, workflow improvements)

## Mandatory Tasks

  - [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by iamr-kumar** [comment]
Hey @CarinaWolli, would like to have your thoughts on this.

Thanks

**Comment by iamr-kumar** [comment]
Hey, I was wondering if someone has taken a look into it.

**Comment by CarinaWolli** [comment]
@joeauyeung or @alannnc can you take a look?

**Review by CarinaWolli — COMMENTED** [review]
Changes work as expected, after my comment is addressed I can approve 🙏

File: packages/features/bookings/lib/handleCancelBooking.ts [hunk 1]
```diff
@@ -730,6 +650,88 @@ async function handleSeatedEventCancellation(
 
   const attendee = bookingToDelete?.attendees.find((attendee) => attendee.id === seatReference.attendeeId);
 
+  if (attendee) {
+    /* If there are references then we should update them as well */
+    const lastAttendee =
```
[the remark this answers is not in the document]
Comment (iamr-kumar) [reply packages/features/bookings/lib/handleCancelBooking.ts]: You are right. `lastAttendee` would never be true here and therefore, the block inside `if(lastAttendee)` would never execute. So I have removed them, as the case for the last attendee gets handled outside of this function.

**Review by CarinaWolli — APPROVED** [review]
Looks good now, thank you your contribution 🙌🏻

**Pull request merged**